mitodl/mitxpro
# #1398 Remove login/register from bulk purchase pages

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1398 

#### What's this PR do?
Removes the login/register menu as well as site-wide notifications on ecommerce bulk pages.

#### How should this be manually tested?
With a site-wide notification that appears on the other pages set up (add a snippet in CMS for this), go to the ecommerce bulk pages (`/ecommerce/bulk/` and `/ecommerce/bulk/receipt/`). Neither the login/register links nor the site-wide notification should show up on the page.
